src/App.tsx

```diff
@@ -10,18 +13,57 @@ import TableContainer from '@mui/material/TableContainer';
 import TableRow from '@mui/material/TableRow';
 import Paper from '@mui/material/Paper';
 import { TableHead } from "@mui/material";
+import { log_levels } from "./schema/Log_Levels.ts";
+
+const apiURL = "/api/views/search/sync";
+const password = "token";
+let username: string
+const query: Query_String = {};
+
+const ACTIONS = {
+  LOGFILTER: "level",
```

Comment:
There are a number of different terms in this codebase that are used interchangeably to describe the log level, off the top of my head...
- Log level
- Level
- Log filter
- Debug level
There may be more.

I don't think it's worth doing here, but could you make an issue to pick a term and apply it across the codebase? Probably the same term that graylog's API uses (level?) to minimise confusion.

Reply:
Created Issue for this #30